src/features/Navigation/MobileDropdown/index.tsx

```diff
@@ -39,7 +39,7 @@ const MobileDropdown: React.FC<Props> = ({ items }) => {
       ref={ref}
       onClick={() => setIsComponentVisible(!isComponentVisible)}
     >
-      <Text variant={'linkMobile'} color={'white'}>
+      <Text variant="link" color="white">
         {t('navigation.mobile.menu')}
       </Text>
       {isComponentVisible ? (
```

Comment:
Its actually this component the mobile-icon thing. 
But as I said, if its not possible to fix before today - lets just merge this.
I know the thing and I'll can look at it in my PR